### PR TITLE
Some fixes need so `sbt run` works again

### PR DESCRIPTION
**Pull request by mkurz**
`sbt run` does not work right now. This PR fixes that :wink:

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -8,11 +8,10 @@ scalaVersion := "2.13.10"
 libraryDependencies ++= Seq(
   guice,
   filters,
+  "ch.qos.logback" % "logback-classic" % "1.3.5",
   "org.sangria-graphql" %% "sangria" % "3.4.1",
   "org.sangria-graphql" %% "sangria-slowlog" % "2.0.5",
   "org.sangria-graphql" %% "sangria-play-json" % "2.0.2",
   "org.scalatest" %% "scalatest" % "3.2.15" % "test")
 
-routesGenerator := InjectedRoutesGenerator
```
Comment (mkurz) [inline build.sbt:16]: This is not needed anymore since Play 2.6 or so.

File: build.sbt [hunk 2]
```diff
@@ -8,11 +8,10 @@ scalaVersion := "2.13.10"
 libraryDependencies ++= Seq(
   guice,
   filters,
+  "ch.qos.logback" % "logback-classic" % "1.3.5",
```
Comment (mkurz) [inline build.sbt:11]: A transitive dependency upgrades slf4j to 2.x, but Play still ships with 1.7 and logback-classic 1.2, which is not compatible with slf4j 2.x yet. Upgraind logback-classic to 1.3 make that work

File: project/build.properties [hunk 3]
```diff
@@ -1 +1 @@
-sbt.version=1.7.3
+sbt.version=1.7.2
```
Comment (mkurz) [inline project/build.properties:1]: Unfortunately sbt 1.7.3 ships with scala-xml 2.x which causes dependency conflicts. Staying on 1.7.2 fixed that.

**Review by sh0hei — APPROVED** [review]
Thanks a lot, @mkurz !

**Pull request merged**